queue_services/entity-filer/src/entity_filer/filing_processors/change_of_directors.py

```diff
@@ -57,7 +57,13 @@ def process(business: Business, filing: Dict):  # pylint: disable=too-many-branc
         if 'appointed' in new_director['actions']:
 
             # add new diretor party role to the business
-            new_director_role = create_director(director_info=new_director)
+            party = create_party(party_info=new_director)
+            role = {
+                'roleType': 'director',
```

Comment:
Should this be 'Director' to match the key in JSON_ROLE_CONVERTER. Not sure whether get by key is case sensitive

Reply:
it should probably be the Enum value?

Reply:
Oh no actually it needs to be 'Director' -> that's the value the FE decided to use for this roletype and in the util I convert it into the enum value (that's why I made the JSON_ROLE_CONVERTER)